main.cpp

```diff
@@ -20,12 +20,24 @@
 #if TARGET_UBLOX_EVK_ODIN_W2
 #include "OdinWiFiInterface.h"
 OdinWiFiInterface wifi;
+
+#elif TARGET_NUCLEO_L476RG
+#include "ESP8266Interface.h"
+// ESP8266_WIFI TX to be connected to D2 (Serial1_Rx)
+// ESP8266_WIFI_RX to be connected to D8 (Serial1_Tx)
+ESP8266Interface wifi(D8, D2);
+
+#elif  TARGET_NUCLEO_F429ZI
```

Comment:
> @MarceloSalazar do you know how to do mbed_apps.json modification ? I did not find it.

You can see the mbed_app.json from the Client example here:
https://github.com/ARMmbed/mbed-os-example-client/blob/master/mbed_app.json#L21